### DISCUSSION TITLE
Why wasn't CLTL2 integrated into sources?

### DISCUSSION THREAD

**Discussion by masinter** [Interlisp/medley#95] [Implementation history]
No demand? Out of funds? Didn't work well enough?

**Comment by Anzus** [comment]
Ran out of time, basically. Wasn't a high enough priority, and the two people working on CLtL2 stuff (Bob and I) moved to other employers and so it was part-time.

**Comment by johnwcowan** [comment]
Here is a [very rough list of places where CLtL2 and ANSI diverge](http://web.archive.org/web/20060111013153/http://www.ntlug.org/~cbbrowne/commonlisp.html#AEN10329); it is intended for marking up a physical copy of CLtL2.  For example, after CLtL2 was published, ANSI changed *-SETF-METHOD to *-SETF-EXPANSION. 

Anything on this list that is already present in Medley's CLtL2 package should be corrected or adjusted accordingly.  Other list items can be ignored until we are ready for a full push towards ANSI compliance, in which case the Hyperspec should  be our guide.
